uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -398,11 +398,18 @@ the UAA Introspect Token endpoint to validate that the tokens have not been revo
 (e.g. a few minutes), and when needed, revoke the more long-lived refresh tokens so that they
 may no longer be used to obtain refreshed access tokens.
 
-When a refresh token in the opaque format is successfully passed to the ``/revoke`` endpoint,
+When a refresh token is successfully passed to the ``/revoke`` endpoint,
```

Comment:
Now I think that I found a better wording for this.

> When the `/revoke` endpoint is successfully invoked with a refresh token...

Because the successfulness should be about the endpoint invocation, not the passing of the token. One can successfully pass a token, and make an unsuccessful request with it.

Reply:
change pushed